release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -162,6 +162,12 @@ You are no longer required to configure AWS VPC endpoints when installing a rest
 
 For more information, see xref:../installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc#installation-custom-aws-vpc-requirements_installing-restricted-networks-aws-installer-provisioned[Requirements for using your VPC].
 
+[id="ocp-4-11-capabilities"]
+==== Additional customization when installing {product-title}
+{product-title} 4.11 allows you to disable the installation of the `baremetal` and `marketplace` Operators, and the `openshift-samples` content that is stored in the `openshift` namespace. You can disable these features by setting the `baselineCapabilitySet` and `additionalEnabledCapabilities` parameters in the `install-config.yaml` configuration file prior to installation. If you disable any of these capabilities during the installation, you can enable them after the cluster is installed. Once a capability has been enabled, it cannot be disabled again.
```

Comment:
Maybe instead of
`You can disable these features by setting the baselineCapabilitySet and additionalEnabledCapabilities parameters in the install-config.yaml`, 
Try:
`You can disable these features by adding the baselineCapabilitySet and additionalEnabledCapabilities parameters to the install-config.yaml`

`setting` makes it sounds like its going to say `setting to something`
Just a suggestion though, up to you!

Reply:
Good suggestion, "adding" sounds better to me.